### DISCUSSION TITLE
How to fix <img=20> issue with ground items??

### DISCUSSION THREAD

**Discussion by BarbarianpriestessAki** [runelite/runelite#16673] [Q&A]
I've tried to disable plugins to see which one it was, but couldnt find it. tried looking up a solution but found no one else seems to have this issue. is there anyway i can fix this? i dont think its any plugins conflicting with one another and im at an absolute loss....

**Comment by Nightfirecat** [comment]
I have no idea what you're talking about. What `<img=20>` issue?

**Reply by BarbarianpriestessAki** [reply]
![image](https://github.com/runelite/runelite/assets/136507359/f18cd08b-04ef-417c-b341-987ef93174a3)
this issue. idk how or what could be conflicting it but i dont have the option of opening runelite in safe mode. im stumped on how to fix it

**Reply by Felanbird** [reply]
> but i dont have the option of opening runelite in safe mode 

Update your launcher, everyone has the option to run safe-mode, it's just some launcher versions are older than the command.

Also you were already in the discord, why didn't you just post your issues with safe-mode there, we could have instructed you how to resolve it.

**Reply by BarbarianpriestessAki** [reply]
i downloaded through the jagex launcher. is there a site i can download it from? i already went to the discord for help with the issue im talking about and came here since ground items doesnt have a question mark thing for it

**Comment by Felanbird** [comment]
How to access safe-mode while using the Jagex Launcher was explained in discord and the issue as of now is believed to be the hub plugin ~~`Equipment Inspector`.~~

As mentioned below it's probably actually `f2p highlight`

**Comment by raiyni** [comment]
Disable `F2P Highlight` probably
